research-hub-web/src/index.html

```diff
@@ -12,21 +12,25 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 
     <!-- Twitter Card -->
-    <meta name="twitter:card" content="summary" />
-    <meta name="twitter:title" content="ResearchHub" />
-    <meta name="twitter:description" content="The ResearchHub connects you with people, resources, and services from across the University to enhance and accelerate your research." />
-    <meta name="twitter:image" content="https://images.ctfassets.net/vbuxn5csp0ik/5IfxoF4WQ4Ks71q2HeLbgC/d0c3bb826ceebd31be789a73edac48fe/UoA_Details_230-min.jpeg?w=120&h=120&fm=png&fit=thumb&f=face" />
+    <meta name="twitter:card" content="summary"/>
+    <meta name="twitter:title" content="ResearchHub"/>
+    <meta name="twitter:description" content="The ResearchHub connects you with people, resources, and services from across the University to enhance and accelerate your research."/>
+    <meta name="twitter:image" content="https://images.ctfassets.net/vbuxn5csp0ik/5IfxoF4WQ4Ks71q2HeLbgC/d0c3bb826ceebd31be789a73edac48fe/UoA_Details_230-min.jpeg?w=120&amp;h=120&amp;fm=png&amp;fit=thumb&amp;f=face"/>
```

Comment:
Just out of curiosity, what are those `amp;`s for?

Reply:
Good question... I'm trying to remember back to when I added that in but I can't remember doing anything specific with the url format?? It does seem to work fine when you post the ResearchHub link on Twitter:
![image](https://user-images.githubusercontent.com/31844476/150713516-c0c85966-9d42-4f6e-a1f5-cccd8febd3a9.png)